.github/actions/rust/action.yml

```diff
@@ -52,11 +52,13 @@ runs:
     # See https://corrode.dev/blog/tips-for-faster-ci-builds/
     - name: Set up build environment
       shell: bash
+      env:
+        TARGETS: ${{ inputs.targets }}
       run: |
         {
           echo "CARGO_PROFILE_RELEASE_LTO=true"
           echo "CARGO_PROFILE_RELEASE_CODEGEN_UNITS=1"
-          if [[ "${{ runner.os }}" == "Linux" && "${{ inputs.targets }}" == "" && "$(command -v mold)" ]]; then
+          if [[ "${{ runner.os }}" == "Linux" && "$TARGETS" == "" && "$(command -v mold)" ]]; then
```

Comment:
For my own understanding, why is `TARGET` better than `inputs.targets`?

Reply:
`zizmor` thinks it allows code injection.

Reply:
Weird. Ok. Thanks for clarifying.